stable/chronograf/values.yaml

```diff
@@ -12,7 +12,7 @@ service:
 ## Persist data to a persitent volume
 persistence:
   enabled: false
-  storageClass: generic
+  # storageClass: <storage class>
```

Comment:
Remove `<storage class>` (i.e. leave the value blank) to be consistent with other charts
Could we add the documentation comments too?